Chapters/2-5-Chapter12.md

```diff
@@ -0,0 +1,175 @@
+## Chapter 12
+
```

Comment:
Oh I actually like the scene jumps in the chapter. It's like you skip over the boring stuff.